Source/plugins/IShell.h

```diff
@@ -306,6 +306,12 @@ namespace PluginHost {
 
     private:
         void* Root(uint32_t& pid, const uint32_t waitTime, const string className, const uint32_t interface, const uint32_t version = ~0);
+
+        /* @stubgen:omit */
+        virtual std::vector<string> GetAllLibrarySearchPaths(const string& locator) const
```

Comment:
Why not make this pure vritual? That makes it mandatory to implement and an empty set adds no value at all, I think..